frontend/javascripts/viewer/model/reducers/update_action_application/volume.ts

```diff
@@ -89,10 +97,57 @@ function applySingleAction(
         ua,
       );
     }
-    case "updateSegmentGroupsExpandedState":
+    case "updateSegmentGroupsExpandedState": {
+      const { areExpanded, groupIds, actionTracingId } = ua.value;
+      const { segmentGroups } = getVolumeTracingById(state.annotation, actionTracingId);
+      const currentlyExpandedSegmentGroupIds = new Set(
+        Object.values(segmentGroups)
+          .filter((g) => g.isExpanded)
+          .map((g) => g.groupId),
+      );
+      const actionGroupIds = new Set(groupIds);
```

Comment:
adding "set" to the name makes the semantics of the variable way clearer than adding "action".

```suggestion
      const groupIdSet = new Set(groupIds);
```

needs to be renamed in the other spots, too.